website/docs/getting-started/project-setup.md

```diff
@@ -22,25 +21,21 @@ You can check your toolchain version using `rustup show` (under "active toolchai
 
 Rust can compile source codes for different "targets" (e.g. different processors). The compilation target for browser-based WebAssembly is called "wasm32-unknown-unknown".  The following command will add this target to your development environment.
 
-`rustup target add wasm32-unknown-unknown`
+```shell
+$ rustup target add wasm32-unknown-unknown
```

Comment:
All other commands in the docs (as far as I know at least) currently don't use the `$` prefix.
I believe we should keep this consistent, so either it should be removed here or added to the few other commands.
While personally I think it looks better aesthetically to have the line prefix, it does mean that if you use the "copy" button next to the snippet you can no longer just paste the command into a console.